server/static/partials/admin/assignment.submit.html

```diff
@@ -42,9 +42,16 @@
                 type="text" name="token" class="form-control input-lg" autocomplete="on" placeholder="ya29.foobar">
               </div>
             </div>
-
-            <div class="form-group dropzone-files">
+            <div class="form-group">
               <label>Files</label>
+              <select name="type" type="select" class="form-control" ng-model="newSubmission.type" ng-options="option for option in ['Upload', 'Copy From Existing Backup']">
```

Comment:
Possible improvement: Don't use the option for option and just set options in the HTML. From the other PR: 
```
                <select class="form-control" required name="otherTag" ng-model="otherTag">
                  <option value="Total" selected="selected">Total Score</option>
                  <option value="composition">Composition</option>
                  <option value="Partner A">Partner A</option>
                  <option value="Partner B">Partner B</option>
                </select>
```